pyproject.toml

```diff
@@ -299,7 +311,7 @@ select = [
   "SIM",  # flake8-simplify
   "T10",  # flake8-debugger
   "T20",  # flake8-print
-  "TCH",  # flake8-type-checking
+  "TC",   # flake8-type-checking
```

Comment:
`TCH` was renamed to `TC` in Ruff [0.8.0](https://github.com/astral-sh/ruff/releases/tag/0.8.0).